packages/babel-register/src/cache.js

```diff
@@ -2,8 +2,11 @@ import path from "path";
 import fs from "fs";
 import { sync as mkdirpSync } from "mkdirp";
 import homeOrTmp from "home-or-tmp";
+import * as babel from "babel-core";
 
-const FILENAME: string = process.env.BABEL_CACHE_PATH || path.join(homeOrTmp, ".babel.json");
+const env = process.env.BABEL_ENV || process.env.NODE_ENV;
```

Comment:
That looks great! I'll keep an eye out and update this when it gets merged in. Thanks for that